packages/headless/src/database/mongodb/order/index.ts

```diff
@@ -278,6 +280,69 @@ export class OrderDatabase implements IOrderDatabase {
         ...(endDate && { $lte: new Date(endDate) }),
       };
     }
-    return await OrderModel.find(queryParams).skip(skip).limit(limit).sort(sort).lean();
+    return await OrderModel.find(queryParams)
+      .skip(skip)
+      .limit(limit)
+      .sort(sort)
+      .lean();
+  }
+
+  async createReview(review: any): Promise<Review | null> {
+    try {
+      return await ReviewModel.create(review);
+    } catch (err) {
+      console.log(err);
+      return null;
+    }
+  }
+
+  async findReview(query: Record<string, any>): Promise<Review[] | null> {
+    try {
+      return await ReviewModel.find(query).lean().exec();
+    } catch (err) {
+      console.log(err);
+      return null;
+    }
+  }
+
+  async addProductRating(productId: string, rating: number): Promise<boolean> {
+    try {
+      const product = await ProductModel.findOne({ id: productId })
+        .lean()
+        .exec();
+      if (!product) return false;
+
+      let ratingObj = product.rating || {};
```

Comment:
``let ratingObj = product.rating || {};`` this code can be rewritten as
``const ratingObj = product.rating || {[rating]: rating};``